network-api/networkapi/wagtailpages/pagemodels/blog/blog.py

```diff
@@ -288,6 +287,12 @@ def ensure_related_posts(self):
             )
 
     def clean(self):
+
```

Comment:
**Nitpick** A blank line after the start of the function on the other hand is more confusing to me 😅